README.md

```diff
@@ -143,25 +233,27 @@ We run as much as we can on Heroku, because that environment puts devs in direct
 
 The remainder of our environment is hosted in AWS. We make heavy use of EC2, RDS, EBS, and S3.
 
-### Testing & monitoring
+### Testing & Monitoring
 
 We maintain quality with unit and functional testing, and we’re constantly trying to find better ways to test. We strongly prefer frameworks, like Laravel, that have unit testing built in to their core.
```

Comment:
Let's link the [Testing RFC](https://github.com/DoSomething/rfcs/blob/master/004-testing.md) here!